databricks_sql_connector/pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
+name = "databricks-sql-connector"
+version = "1.0.0"
```

Comment:
It should be next major version of the existing `databricks-sql-connector`, I believe it is `4.0.0`

Reply:
I'm actually not so sure about it. I think, if the library behavior remains the same for user - no need to change major version number. At least for now we can keep a current version number, and decide how to increment it before the next release